pre_commit_hooks/pretty_format_json.py

```diff
@@ -90,20 +100,23 @@ def pretty_format_json(argv=None):
     status = 0
 
     for json_file in args.filenames:
-        with open(json_file) as f:
+        with io.open(json_file, encoding='utf-8') as f:
```

Comment:
@asottile I'm assuming all inputs are UTF-8 compatible here. Not sure this is a good way to handle it.